home/jobs/OMERO-test-integration/config.xml

```diff
@@ -185,4 +191,4 @@ deactivate
   <buildWrappers>
     <hudson.plugins.timestamper.TimestamperBuildWrapper plugin="timestamper@1.9"/>
   </buildWrappers>
-</project>
\ No newline at end of file
+</project>
```

Comment:
Unfortunately, Jenkins does revert these so this will create an unnecessary diff in running devspaces. Can this change be removed?